app/src/main/java/com/zeapo/pwdstore/utils/Extensions.kt

```diff
@@ -98,7 +99,7 @@ fun Context.getEncryptedPrefs(fileName: String): SharedPreferences {
 }
 
 @MainThread
```

Comment:
Annotation should `@WorkerThread`

Reply:
Not too sure about that but I'll look into it.

Reply:
Ideally this would not need any kind of thread hint, we should just use withcontext whenever context matters.